src/components/plots/piePlot.ts

```diff
@@ -12,9 +18,15 @@ export module Plots {
    *   "outer-radius" - Accessor determining the distance from the center to the outer edge of the sector
    *   "value" - Accessor to extract the value determining the proportion of each slice to the total
    */
-  export class Pie extends Plot {
+  export class Pie<D> extends Plot {
 
     private _colorScale: Scales.Color;
+    private static _INNER_RADIUS_KEY = "inner-radius";
+    private static _OUTER_RADIUS_KEY = "outer-radius";
+    private static _SECTOR_VALUE_KEY = "sector-value";
+    private _key2DataExtents: D3.Map<any[]>;
```

Comment:
Since this is mirroring the `_projections` objects, we should make it clear:
```
private _propertyExtents: D3.Map<any []>;
private _attrExtents: D3.Map<any []>; // for DOM attributes
```